mlrun/execution.py

```diff
@@ -201,7 +201,10 @@ def out_path(self):
     @property
     def labels(self):
         """Dictionary with labels (read-only)"""
-        return deepcopy(self._labels)
+        labels = deepcopy(self._labels)
+        if not self.is_logging_worker():
+            labels.pop("host", None)
+        return labels
```

Comment:
please explain in a comment why do you pop such labels
and, in addition, maybe add a declaritve list of labels that we dont want to be used/read from non-logging worker
another idea is to have `labels`/`non_logging_worker_labels`